### PR TITLE
Sample list - preliminary chagnes

### PR DESCRIPTION
**Pull request by withrocks**
These are some rather minor changes that need to be in place before a larger patch with support for the new `Substance` type.

### CODE REVIEW

File: src/clims/models/plugin_registration.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from sentry.db.models import (Model, sane_repr)
+
+
+class PluginRegistration(Model):
+    """
+    Lists every registration of a plugin, ensuring that configuration created by a plugin can be traced to a particular
+    version of that plugin.
+
+    Take for example a look at the ItemType model. When a plugin registers a particular
+
```
Comment (GitEdvard) [inline src/clims/models/plugin_registration.py]: Here, the text stops in the middle of a sentence!

File: src/clims/models/plugin_registration.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    If you start the service with a particular version of a plugin (e.g. clims_genetics@1.0.0) and then update
+    it only at 2.0.0, we'll only have a record of those two plugins on the system, even if there are more
+    available on PyPi.
+    """
```
Comment (GitEdvard) [inline src/clims/models/plugin_registration.py:12]: I dont catch the point here.

**Review by GitEdvard — COMMENTED** [review]
Ok, I have comments on the comments. Tell me when you want to merge. 

Is substance going to replace ActivityInstance? Did I got that right?

File: src/clims/models/plugin_registration.py (hunk 1 above)
Comment (withrocks) [reply src/clims/models/plugin_registration.py]: Thanks! That happens once in a while for me.

File: src/clims/models/plugin_registration.py (hunk 2 above)
Comment (withrocks) [reply src/clims/models/plugin_registration.py:12]: I think it's too much detail there anyways. I'll remove it. Also removed that cut sentence.

**Pull request merged**